test/v200/utils/common/test_utils.go

```diff
@@ -0,0 +1,283 @@
+package common
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+	"time"
+
+	schema "github.com/devfile/api/v2/pkg/apis/workspaces/v1alpha2"
+	header "github.com/devfile/api/v2/pkg/devfile"
+)
+
+const (
+	// maxCommands : The maximum number of commands to include in a generated devfile
+	maxCommands = 10
+	// maxComponents : The maximum number of components to include in a generated devfile
+	maxComponents = 10
+	// numDevfiles : the number of devfiles to create for each test
+	numDevfiles = 5
+)
+
+const (
+	defaultTempDir = "./tmp/"
+	logFileName    = "test.log"
+	// logToFileOnly - If set to false the log output will also be output to the console
+	logToFileOnly = true // If set to false the log output will also be output to the console
+)
```

Comment:
```suggestion
const (
	// maxCommands : The maximum number of commands to include in a generated devfile
	maxCommands = 10
	// maxComponents : The maximum number of components to include in a generated devfile
	maxComponents = 10
	// numDevfiles : the number of devfiles to create for each test
	numDevfiles = 5
	defaultTempDir = "./tmp/"
	logFileName    = "test.log"
	// logToFileOnly - If set to false the log output will also be output to the console
	logToFileOnly = true // If set to false the log output will also be output to the console
)
```

Reply:
sometime it can be good to group constants. But will fix it. The max ones will probably become configurable in eth ;ong run anyway,